crates/db/src/lib.rs

```diff
@@ -15,13 +15,14 @@ mod prost_serde;
 
 pub type DBResult<T> = Result<T, RepositoryError>;
 
-#[derive(Debug, Snafu)]
+#[derive(Debug, Error)]
 pub enum RepositoryError {
+    #[error("KeyValueStorage error: {source}")]
     KeyValueStorage {
-        #[snafu(source)]
+        #[source]
         source: Box<dyn std::error::Error>,
     },
-    #[snafu(display("Invalid data: [{}]", source))]
+    #[error("Invalid data: [{source}]")]
     InvalidData { source: Box<dyn std::error::Error> },
```

Comment:
Maybe we can just use plain variant, as it's not required to use structured one for `thiserror`

```suggestion
    InvalidData(Box<dyn std::error::Error>),
```